scr/example-01b_simple-beam-3-span_point-load.py

```diff
@@ -0,0 +1,303 @@
+        shear_vals.append(beam.Members[member].shear("Fy", x))
+        moment_vals.append(beam.Members[member].moment("Mz", x))
+        deflection_vals.append(beam.Members[member].deflection("dy", x))
+
+    return xs, shear_vals, moment_vals, deflection_vals
+
+
+def plt_combined(beam, fpath):
+    """Plot shear force, bending moment and deflection in a single plot.
+
+    Parameters
+    ----------
+    beam : FEModel3D object
+        Beam FEModel3D object
+    fpath : pathlib Path object or str
+        Full path including filename of the figure to save the figure.
+    """
+
+    # figure
+    figscale = 1.0
+    figw = 3.487
+    figh = 3.5 * figw / 1.618
+    figsz = (figscale * figw, figscale * figh)
+    fig, axs = plt.subplots(3, 1, figsize=figsz, sharex=True)
+
+    # data for shear, moment and deflection diagrams
+    xs = []
+    shear_vals = []
+    moment_vals = []
+    deflection_vals = []
+
+    for member in beam.Members:
+        xss, shears, moments, deflections = gen_plt_data(beam, member)
+        xs.append(xss)
+        shear_vals += shears
+        moment_vals += moments
+        deflection_vals += deflections
+
+    # fmt : off
+    # distances along the beam, convert from local coordinates to global coordinates, must be a better way to do this
+    xs = [list(np.diff(x, prepend=0)) for x in xs]
+    xs = np.cumsum(np.concatenate(xs).ravel())
+    # fmt : on
+
+    # plots : shear
+    axs[0].plot(xs, shear_vals)
+    axs[0].plot(
+        [xs[0], xs[0], xs[-1], xs[-1]],
+        [shear_vals[0], 0, 0, shear_vals[-1]],
+        color="k",
+        lw=2.0,
+    )
+    axs[0].grid(visible=True, which="major", axis="both", ls=":", alpha=0.5)
+    axs[0].set_ylabel("Shear Force, $V$ [kip]")
+
+    # moment
+    axs[1].plot(xs, moment_vals)
+    axs[1].plot(
+        [xs[0], xs[0], xs[-1], xs[-1]],
+        [moment_vals[0], 0, 0, moment_vals[-1]],
+        color="k",
+        lw=2.0,
+    )
+    axs[1].grid(visible=True, which="major", axis="both", ls=":", alpha=0.5)
+    axs[1].set_ylabel("Bending Moment, $M$ [kip-in]")
+
+    # deflections
+    axs[2].plot(xs, deflection_vals)
+    axs[2].plot(
+        [xs[0], xs[0], xs[-1], xs[-1]],
+        [deflection_vals[0], 0, 0, deflection_vals[-1]],
+        color="k",
+        lw=2.0,
+    )
+    axs[2].grid(visible=True, which="major", axis="both", ls=":", alpha=0.5)
+    axs[2].set_xlabel("Distance, $x$ [in]")
+    axs[2].set_ylabel("Deflection, $\delta$ [in]")
+
+    fig.savefig(fpath, format="png", dpi=150, bbox_inches="tight")
+
+
+def main():
+    # paths and filenames
+    path_root = Path(r"../")
+    # path_data = path_root.joinpath("data/")
+    path_plt = path_root.joinpath("plt/")
+    path_report = path_root.joinpath("rpt/")
+    fname_vtk = "example-01b-vtk.png"
+    fname_fig = "example-01b.png"
+    fname_rpt = "example-01b_rpt.pdf"
+    ffigvtk = path_plt.joinpath(fname_vtk)
+    frpt = path_report.joinpath(fname_rpt)
+    ffig = path_plt.joinpath(fname_fig)
```

Comment:
I don't like this kind of stuff in the examples. This has to do with preferences for how/where to save the files and very little to do with using PyNite and will turn off a lot of users i they see it.